ui/js/component/common.js

```diff
@@ -68,23 +68,32 @@ export class CreditAmount extends React.PureComponent {
     isEstimate: React.PropTypes.bool,
     label: React.PropTypes.bool,
     showFree: React.PropTypes.bool,
+    showFullPrice: React.PropTypes.bool,
     look: React.PropTypes.oneOf(["indicator", "plain"]),
   };
 
   static defaultProps = {
     precision: 2,
     label: true,
-    showFree: false,
     look: "indicator",
+    showFree: false,
+    showFullPrice: false,
   };
 
   render() {
     const minimumRenderableAmount = Math.pow(10, -1 * this.props.precision);
-    const { amount, precision } = this.props;
+    const { amount, precision, showFullPrice } = this.props;
 
-    let formattedAmount = amount > 0 && amount < minimumRenderableAmount
-      ? "<" + minimumRenderableAmount
-      : formatCredits(amount, precision);
+    let formattedAmount;
+    let fullPrice = formatCredits(amount, 5);
```

Comment:
What if the price is less than 0.00001?

This logic probably needs to be more dynamic.

Reply:
@kauffj  check this: https://gist.github.com/btzr-io/e9a2b0f8cf5dcde3a0ee0ec2f0a0aa15,

Reply:
need some feedback / confirmation to implement that :stuck_out_tongue:  ^^